cmake/llvm-project.cmake

```diff
@@ -61,6 +61,7 @@ list(APPEND LLVM_INCLUDE_DIRS
 
 # Linker flags
 set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,-rpath -Wl,${LLVM_BINARY_DIR}/llvm/lib")
+list(APPEND CMAKE_BUILD_RPATH "${LLVM_BINARY_DIR}/llvm/lib")
```

Comment:
Can we double check the LLVM files to see what impact this has on our build?

Reply:
This setting is the proper way to encode the RPATHs into the binaries for BUILD. I have removed the previous line as it is no longer necessary.

We may need to set CMAKE_INSTALL_RPATH for installation runtime paths.